build/suite-router.sh

```
@@ -0,0 +1,221 @@
+        suites+=("$input")
+      fi
+      ;;
+  esac
+}
+
+# infer suggestions from changed files (never used to widen the gate)
+infer_suggestions() {
+  [ -z "${CHANGED_FILES:-}" ] && return
+  while IFS= read -r f; do
+    case "$f" in
+      internal/router/*|internal/ext_proc/*)
```

Shellcheck (0.11.0)

[warning] This pattern always overrides a later one on line 83.

(SC2221)